lib/compat/wordpress-6.3/navigation-block-preloading.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Patches resources loaded by the block editor page
+ * to include Navigation posts.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Preloads requests needed for Navigation posts
+ *
+ * @param array $preload_paths    Preload paths to be filtered.
```

Comment:
We seem to be missing inline docs for the `context` argument.

Reply:
✅